tests/integration/ceph_object_test.go

```diff
@@ -102,6 +106,16 @@ func (s *ObjectSuite) TestWithTLS() {
 	runObjectE2ETest(s.helper, s.k8sh, s.Suite, s.settings.Namespace, tls)
 }
 
+func (s *ObjectSuite) TestWithBrokenTLS() {
```

Comment:
Secondly I don't fully understand the difference in how testing is performed  `TestWithBrokenTLS()`  and `TestWithTLS()` . The only notable difference is prefix value, Am I missing something?

Reply:
Correct, the prefix breaks the server cert validation by the client since the DNS is not registered.